water_monitoring_system/config/settings/test.py

```diff
@@ -0,0 +1,51 @@
+"""
+With these settings, tests run faster.
+"""
+
+from .base import *  # noqa
+from .base import env
+
+# GENERAL
+# ------------------------------------------------------------------------------
+# https://docs.djangoproject.com/en/dev/ref/settings/#secret-key
+SECRET_KEY = env(
+    "DJANGO_SECRET_KEY",
+    default="62m3zMPL91SKFFEQr4CgueM1UjmQAxlBgXF24qdnacyuALmnAq8Nh0uBI4s2VkOb",
```

Comment:
Seems like default `SECRET_KEY` is not sensitive.

Please verify @PragatiVerma18 @ramanaditya

Reply:
No this is not sensitive